docs/raspberry_pi4.md

```diff
@@ -22,19 +19,43 @@ make custom_all FEATURES=u_boot
 4. Set configuration for Raspberry Pi 4: `make rpi_4_defconfig`
 5. Build: `make`
 6. Copy u-boot.bin
+7. Create boot.scr with boot script like below: `tools/mkimage -A arm64 -T script -C none -d boot.txt boot.scr`
+
+```
+fatload mmc 0:1 ${kernel_addr_r} EFI/BOOT/BOOTAA64.EFI
```

Comment:
Does the `mmc 0:1` depend on user's environment? If so, adding a note about it might be good.

Reply:
It is from Raspberry Pi OS's partition layout.
Partition 0 is bootfs and 1 is rootfs.
This document assumed using Raspberry Pi OS.